modules/libcom/src/as/asLib.h

```diff
@@ -201,7 +239,10 @@ typedef struct{
     void            *rpcl;
     ELLLIST         uagList; /*List of ASGUAG*/
     ELLLIST         hagList; /*List of ASGHAG*/
+    ELLLIST         methodList; /*List of ASGMETHOD*/
+    ELLLIST         authList;   /*List of ASGAUTHORITY*/
     int             trapMask;
+    int             isTLS; // -1 not set, 0 false, 1 = true
```

Comment:
I'm not sure I follow what a "TLS" condition is meant to capture, and how it would be used.  Maybe an example would help?